### PR TITLE
Update create-custom-role.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated view reports on the document

fixes#https://github.com/MicrosoftDocs/memdocs/issues/3577

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: memdocs/intune/fundamentals/create-custom-role.md [hunk 1]
```diff
@@ -164,6 +164,7 @@ The following permissions are available when creating custom roles.
 | Mobile apps/Read | View mobile applications such as store apps, line-of-business apps, web-links or built-in apps. You can also view books purchased through the Apple Volume Purchase Program or add eBook categories. You can view iOS VPP Tokens, Windows Symantec certificates, Windows side loading keys, app categories, or the Android for Work connection. | 
 | Mobile apps/Relate | Create relationships with other managed apps using Dependencies and Supersedence features. Without this permission, IT admins are not able to add App dependency or supercedence relationships when creating or editing Win32 apps. |
 | Mobile apps/Update | Manage mobile applications such as store apps, line-of-business apps, web-links or built-in apps. You can also manage books purchased through the Apple Volume Purchase Program or add eBook categories. You can manage iOS VPP Tokens, Windows Symantec certificates, Windows side loading keys, app categories, or the Android for Work connection. | 
+|Mobile apps\View reports| View Reports on mobile applications such as store apps, line-of-business apps, web-links or built-in apps. |
```
Comment (JohanFreelancer9) [inline memdocs/intune/fundamentals/create-custom-role.md:167]:
```suggestion
|Mobile apps/View reports| View reports on mobile applications such as store apps, line-of-business apps, web links, and built-in apps. |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Smritib17 Please review and signoff this PR. Thanks

**Comment by Smritib17** [comment]
#sign-off

**Pull request merged**